### DISCUSSION TITLE
Logged out of all admin accounts

### DISCUSSION THREAD

**Discussion by TubHat** [Pryaxis/TShock#2381] [General]
So i did that /register command in my terraria world and used the /setup command hoping i would permanently be logged in as an admin in the game but apparently when i logged in, my admin privelage was gone and i couldn't use the /setup command again as it said that the /setup command is disabled pls help
![image_2021-06-26_182530](https://user-images.githubusercontent.com/86521292/123513664-e3879f80-d6ab-11eb-8239-4fb58eb4840a.png)

**Comment by hakusaro** [comment]
Check the instructions over here: https://github.com/Pryaxis/TShock/discussions/2325#discussioncomment-772279

Let me know if that doesn't help?
